bears/general/AnnotationBear.py

```diff
@@ -88,66 +96,111 @@ def find_annotation_ranges(self,
             A dictionary containing the various ways to define multi-line
             comments in a language.
         :return:
-            Two tuples first containing a tuple of strings, the second a tuple
-            of comments.
+            Four tuples containing namedtuples for singleline strings,
+            multiline strings, singleline comments and multiline comments
+            respectively.
         """
         text = ''.join(file)
-        strings_range = []
-        comments_range = []
+        singleline_string_range = []
+        multiline_string_range = []
+        singleline_comment_range = []
+        multiline_comment_range = []
+        fields = ('start_delimiter_range end_delimiter_range'
+                  ' content_range full_range')
+        singleline_string = namedtuple('singleline_string', fields)
```

Comment:
http://stackoverflow.com/questions/16377215/how-to-pickle-a-namedtuple-instance-correctly